app/components/NestedCheckboxes.tsx

```diff
@@ -34,9 +34,8 @@ function NestedCheckboxNode({
   )
   const [topLevelValue, setTopLevelValue] = useState(false)
   const [childValues, setChildValues] = useState(
-    new Array<boolean>(nodes.length).fill(false)
+    nodes.map((node) => node.defaultChecked || false)
   )
-
```

Comment:
Restore newline.

Reply:
Not sure how that happened, resolving now